writeup/workflow.cwl

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env cwl-runner
+
+# INPUTS:
+#   submission_id: Submission ID
+#   synapse_config: filepath to .synapseConfig file
+#   admin_folder_id: Synapse Folder ID accessible by an admin
+#   submitter_folder_id: Synapse Folder ID accessible by the submitter
+#   workflow_id: Synapse File ID that links to the workflow archive
+
+cwlVersion: v1.0
+class: Workflow
+
+label: MICCAI 2024 MIDI-B workflow to accept challenge writeups
+doc: >
+  This workflow will validate a participant's writeup, checking for:
+    - Submission is a Synapse project
+    - Submission is not the challenge site (which is a Synapse project)
+    - Submission is accessible to the admin team
+  Archive (create a project copy) if the submission is valid.
+
+requirements:
+  - class: StepInputExpressionRequirement
+
+inputs:
+  - id: submissionId
+    type: int
+  - id: synapseConfig
+    type: File
+  - id: adminUploadSynId
+    type: string
+  - id: submitterUploadSynId
+    type: string
+  - id: workflowSynapseId
+    type: string
+  - id: admin
+    type: string
+    default: "mdsage1"  # TODO: enter admin username (they will become the archive owner)
```

Comment:
```suggestion
    default: "MIDI-B De-identification Organizers"
```

That way, it will check that the writeup is accessible to Organizers team. If not, the participant will get notified to share it with "MIDI-B De-identification Organizers" team